packages/blockchain-wallet-v4-frontend/src/scenes/plugin/Activity/TransactionList/TransactionListItem.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import styled from 'styled-components'
+
+import { getShortTime } from '@core/transactions/eth'
+import { EthProcessedTxType } from '@core/transactions/types'
+import { Color, Text } from 'blockchain-info-components'
+import CryptoAddress from 'components/CryptoAddress/CryptoAddress'
+import CoinDisplay from 'components/Display/CoinDisplay'
+import FiatDisplay from 'components/Display/FiatDisplay'
+import { Expanded, Flex } from 'components/Flex'
+
+import TransactionIcon from '../TransactionIcon/TransactionIcon'
+import StatusText from './StatusText'
+
+export interface ITransactionListItemProps {
+  coin: string
+  item: EthProcessedTxType
+  onClick(tx: EthProcessedTxType): void
+  status: 'CONFIRMED' | 'PENDING'
+}
+
+const ItemWrapper = styled.div`
+  padding-block: 10px;
+  text-decoration: none;
+  cursor: pointer;
+
+  &:hover {
+    background-color: ${Color('exchangeLogin')};
+  }
+`
+
+const getTransactionTitle = (type: string, coin: string) => {
```

Comment:
why is it outside of component body?

Reply:
It's just a util function (this one and the ones that you mentioned below). In my opinion make no sense to put such functions inside component's body - it will be recreated at each rerender. Also will be easier to extract it to a different file is it will be needed.